docs/src/index.md

```diff
@@ -35,6 +35,23 @@ When displayed, these three images look like this:
 <p>
 ```
 
+```
+
+Drawing a Rectangle.
+
+```@example usage
+using TestImages, ImageDraw, ImageCore
+using FileIO # hide
+img = testimage("lighthouse")
+img_example = draw!(img, Polygon(RectanglePoints(Point(1, 1), Point(5, 9))), RGB{N0f8}(1))
+save("images/lighthouse_rectangle.png", img_example) # hide
```

Comment:
IIUC the `example` block automatically inserts the result of the last expression so you don't need to save it. 

```suggestion
using TestImages, ImageDraw, ImageCore, ImageShow
img = testimage("lighthouse")
img_example = draw!(img, Polygon(RectanglePoints(Point(1, 1), Point(5, 9))), RGB{N0f8}(1))
```

`ImageShow` is required to reduce the displayed image size because otherwise it will save images using SVG format, which is crazy for normal image types.